### ISSUE TITLE
Contributions -  Contribution type add - required combo boxes not marked as required

### ISSUE THREAD

**Issue by aviannamiller** [omeka/plugin-Contribution#82]
"Description: On the "Contributions" page, in the contribution type add section, there are two combo boxes that are required but are not marked as such, the "item type" combo box and the ""elements"" combo box, are not shown to be required fields.  

The "Elements" combo box is announced as required but not visually shown to be required. 

Note: the "Question" edit field in the contributor question section is also not marked as required, nor is it announced as required. 

Expected Results: Components that require user input should be visually shown and announced by screen readers 

Actual Results: Required combo boxes do not announce that they are required and have no visual indication of being required"

3.3.2 Labels or Instructions; Level A; WCAG 2.0
1.3.1 Info and Relationships; Level A; WCAG 2.0
